apps/expo/src/components/toggle/Toggle.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { render } from '@testing-library/react-native'
+import Toggle from '~/components/toggle/toggle'
```

Comment:
please remove this test file...there is already one that more rigid written for you under __tests__ folder

![image](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/5f1e876c-8364-431e-be6f-baec8d30a654)

Reply:
Removed